src/App.vue

```diff
@@ -742,6 +745,7 @@ export default class App extends Mixins(
       filingSubType: data.type,
       isFutureEffective: header.isFutureEffective,
       name: header.name,
+      paymentStatusCode: header.paymentStatusCode,
```

Comment:
Are you trying to handle a "paid", rejected authorization here? That's an invalid scenario so you can undo this code.